### DISCUSSION TITLE
Switch from internal EJBCA MariaDB with Galera cluster to external Postgres/Microsoft SQL DB service in Azure

### DISCUSSION THREAD

**Discussion by bergstiefelbene** [Keyfactor/ejbca-ce#171] [General]
Hi,
we want to improve our Azure VM EJBCA two node design with internal Maria DB´s running in a Galera cluster to an external Azure database service using either Microsoft SQL database or Postgres also in a cluster.

My question is can we run this design using different database vendors with the installed Azure marketplace EJBCA VM´s ( OracleLinux) ?
If yes, is there a "db migration" script or information available?

**Comment by primetomas** [comment]
EJBCA Enterprise have a database command line that can be used to migrate from one database vendor to another. The snag is of course that the different databases have different column types so it's not possible to export/import between different vendors.

If you are using Keyfactor's Azure Marketplace offering you have an SLA and can contact support for more help.

Cheers,
Tomas
